vhdbuilder/packer/test/linux-vhd-content-test.sh

```diff
@@ -484,7 +484,7 @@ testCoreDumpSettings() {
   #   A section heading -- this file is only supposed to have '[Coredump]'
   #   Settings, which take the form 'NAME=VALUE', where values can be empty or strings, and strings
   #   is pretty loose (more or less any printable character).
-  testSettingFileFormat $test $settings_file '^(#|$)' '^\[Coredump\]$' '^[A-Za-z_]+=[^[:cntrl:]]*$'
```

Comment:
please revert this. the casing is correct pre-diff, looks like merge/rebase issue.

Reply:
done, thank you for catching this, bad rebase!